src/rqt_pose_view/pose_view_widget.py

```diff
@@ -82,6 +82,14 @@ def __init__(self, plugin):
         self._update_timer.timeout.connect(self.update_timeout)
         self._update_timer.start(40)
 
+        # subscribe immediately if topic name found on param server
+        if rospy.has_param('~topic'):
+            topic = rospy.get_param('~topic')
```

Comment:
Since this parameter is set on the `rqt` Python node and therefore shared across all Python plugin it might be good to choose a more specific name like `pose_view_topic`.